backend/curriculum_tracking/api_views.py

```diff
@@ -145,7 +145,7 @@ class CardSummaryViewset(viewsets.ModelViewSet):
         | core_permissions.HasObjectPermission(
             permissions=Team.PERMISSION_VIEW,
             get_objects=core_permissions.get_teams_from_user_filter("assignees"),
-        ),
+        )
```

Comment:
What autoformatted are you using? It looks like your setup is disagreeing with mine.

This project uses `black`, just the default settings. 

It's important that we use the same setup.